mmdet/datasets/xml_style.py

```diff
@@ -26,8 +27,16 @@ def load_annotations(self, ann_file):
             tree = ET.parse(xml_path)
             root = tree.getroot()
             size = root.find('size')
-            width = int(size.find('width').text)
-            height = int(size.find('height').text)
+            width = 0
+            height = 0
+            if size is not None:
+                width = int(size.find('width').text)
+                height = int(size.find('height').text)
+            else:
+                imgname = osp.join(self.img_prefix, 'JPEGImages',
+                                '{}.jpg'.format(img_id))
+                img = Image.open(imgname)
+                width,height = img.size
```

Comment:
Please fix the linting error. For the code style, you may refer to the [contributing guide](https://github.com/open-mmlab/mmdetection/blob/master/.github/CONTRIBUTING.md#code-style).

Reply:
Done thanks